internal/apiserver/draft_test.go

```diff
@@ -128,3 +192,30 @@ func TestDraft(t *testing.T) {
 		assert.Equal(t, response.Body.String(), "[]")
 	})
 }
+
+func TestDraftsLimit(t *testing.T) {
+	t.Cleanup(testutil.CleanDb(ctx, t, a.db.Connector))
+	t.Cleanup(testutil.CleanCache(ctx, t, a.cache))
+	t.Cleanup(testutil.CleanStorage(t, a.storage))
+
+	user := testutil.CreateUser(ctx, t, a.db.Connector, "")
+	sessionToken := testutil.CreateToken(ctx, t, a.cache, user)
+
+	for i := 0; i < 5; i++ {
```

Comment:
i think 5 is unnecessary high number, if there is no other reason let's lower it (in config as well)